plugins/chat/assets/javascripts/discourse/models/chat-message-draft.js

```diff
@@ -0,0 +1,62 @@
+import { tracked } from "@glimmer/tracking";
+
+export default class ChatMessageDraft {
+  static create(args = {}) {
```

Comment:
Why bother with this when we can just do `new ChatMessageDraft`?

Reply:
It's mostly for history, we have always created our models this way and I think it's nice, it has some similarities with rails too. You can easily created dedicated creator if needed too.